### PR TITLE
[chore] enabled correct localisation of storage provider types

### PR DESCRIPTION
**Pull request by Kharonus**
### What?

- fixed storage list component to render a localized provider type string
- increased column width of provider type column

#### Before

image

#### After

image

### CODE REVIEW

File: modules/storages/app/components/storages/admin/storage_list_component.html.erb [hunk 1]
```diff
@@ -41,7 +41,7 @@
             end
 
             grid.with_area(:provider, tag: :div, color: :subtle, mr: 3, hide: :sm, data: { 'test-selector': 'storage-provider' }) do
-              render(Primer::Beta::Truncate.new(font_weight: :light)) { storage.short_provider_type.capitalize }
+              render(Primer::Beta::Truncate.new(font_weight: :light)) { I18n.t("storages.provider_types.#{storage.short_provider_type}.name") }
```
Comment (akabiru) [inline modules/storages/app/components/storages/admin/storage_list_component.html.erb:44]: Nice catch 🍻

File: modules/storages/app/components/storages/admin/storage_list_component.sass [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .op-storage-list--header,
 .op-storage-list--row
   display: grid
-  grid-template-columns: 1fr 180px 100px 240px
+  grid-template-columns: 1fr 180px 160px 240px
```
Comment (akabiru) [inline modules/storages/app/components/storages/admin/storage_list_component.sass:4]: 🟢  Perhaps we should even go for a full fractional measurement system 🤔

**Review by akabiru — APPROVED** [review]

File: modules/storages/app/components/storages/admin/storage_list_component.html.erb (hunk 1 above)
Comment (dominic-braeunlein) [reply modules/storages/app/components/storages/admin/storage_list_component.html.erb:44]: 👍

File: modules/storages/app/components/storages/admin/storage_list_component.sass (hunk 2 above)
Comment (dominic-braeunlein) [reply modules/storages/app/components/storages/admin/storage_list_component.sass:4]: The new width works 👍

Comment (dominic-braeunlein) [reply modules/storages/app/components/storages/admin/storage_list_component.sass:4]: > 🟢 Perhaps we should even go for a full fractional measurement system 🤔

@akabiru  Do you know how the tables are supposed to react to smaller widths? Header and content are not aligned on smaller screens.

storages-responsive-table

**Review by dominic-braeunlein — APPROVED** [review]

File: modules/storages/app/components/storages/admin/storage_list_component.sass (hunk 2 above)
Comment (akabiru) [reply modules/storages/app/components/storages/admin/storage_list_component.sass:4]: > > 🟢 Perhaps we should even go for a full fractional measurement system 🤔
> 
> @akabiru Do you know how the tables are supposed to react to smaller widths? Header and content are not aligned on smaller screens.
> 
> storages-responsive-table

Good eye @dominic-braeunlein 👍🏾 we're solving part of the resize issues in https://community.openproject.org/wp/50307 I believe @Kharonus also has some ideas on the header and content mis-alignment. The discussions on the "primer like" table are WIP as primer doesn't offer a _real_ table yet.

The topic could use your insight for sure 😎

Comment (Kharonus) [reply modules/storages/app/components/storages/admin/storage_list_component.sass:4]: @akabiru already answered it, the sizing is fixed in a different wp. 

regarding fractional column width: I didn't want to change current behavior too much, as I was not part of the latest discussion about the general behavior of that grid.

**Comment by Kharonus** [comment]
I'll merge this fix, as the resizing issue and related to that any restructuring of the grid will take place in [OP#50307](https://community.openproject.org/wp/50307)

**Pull request merged**